cmd/create/cluster/cmd.go

```diff
@@ -156,6 +157,9 @@ var args struct {
 	hostedClusterEnabled bool
 	billingAccount       string
 
+	// Cluster Admin
+	clusterAdminUser     string
+	clusterAdminPassword string
```

Comment:
NP: Could we put a newline after this? Just to separate from the Audit Log Forwarding var

Reply:
good point, fixed